src/js/jsx/sections/pages/LayerFace.jsx

```diff
@@ -40,16 +40,48 @@ define(function (require, exports, module) {
         system = require("js/util/system"),
         strings = require("i18n!nls/strings");
 
+    /*
+     * Function for checking whether React component should update
+     * Passed to Droppable composed component in order to save on extraneous renders
+     *
+     * @param {Object} nextProps - Next set of properties for this component
+     * @return {boolean}
+     */
+    var shouldComponentUpdate = function (nextProps) {
```

Comment:
Now you're using this as a component method, but it's defined separately for no apparent reason. Unless there _is_ some reason, it should be moved inline with all the other methods for consistency.